recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/wayland/weston_%.bbappend

```diff
@@ -31,7 +31,7 @@ do_install_append_r8a7795() {
         sed -e '$a\\' \
             -e '$a\[output]' \
             -e '$a\name=HDMI-A-1' \
-            -e '$a\transform=270' \
+            -e '$a\transform=0' \
```

Comment:
Do we need to do the same for r8a7796?

Reply:
According to prod-devel requirements we need to support M3.

Reply:
Updated